INSTALL.md

```diff
@@ -1,43 +1,139 @@
 # 1. Back-end setup
```

Comment:
The `h1` should be something like "Installation instructions" and the following heading levels should be adjusted accordingly.